lldb/include/lldb/lldb-enumerations.h

```diff
@@ -135,6 +135,9 @@ FLAGS_ENUM(LaunchFlags){
 /// Thread Run Modes.
 enum RunMode { eOnlyThisThread, eAllThreads, eOnlyDuringStepping };
 
+/// Execution directions
+enum RunDirection { eRunForward, eRunReverse };
+
```

Comment:
If we don't add an overload to continue by adding `SBProcess::ReverseContinue()` then this should be kept internal and not in this header file. If you add a new `SBProcess::Continue(lldb::RunDirection)` overload then this is needed. I would prefer a dedicated `SBProcess::ReverseContinue()` function.